src/numberApp.js

```diff
@@ -0,0 +1,55 @@
+import numberGame from './numberGame.js';
+
+const button = document.getElementById('guessButton');
+const numberInput = document.getElementById('input');
+const trysLeft = document.getElementById('trysLeft');
+const feedback = document.getElementById('feedback');
+const winLose = document.getElementById('winLose');
+let trys = 4;
+
+
+//random integer syntax pulled from MDN https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+function getRandomInt(max) {
+    return Math.floor(Math.random() * (max - 1)) + 1; 
+}
+let randomNum = getRandomInt(20); 
+
+
+button.addEventListener('click', () => {
+    trysLeft.textContent = 'Trys Left ' + trys;
+    trys -= 1;
+    trysLeft.textContent = 'Trys Left ' + trys;
+    let youLost = 'You Lost :(';
+
+    if(trys === 0) {
+        winLose.textContent = youLost;
+        button.disabled = true;
+    }
+    
+    let numGrab = numberInput.value;
+    
+    const compResult = numberGame(numGrab, randomNum);
+    let feedString = ' ';
+    
+    if(compResult === -1) {
+        feedString = 'Guess is too low!';
+        feedback.textContent = feedString;
+        return;
```

Comment:
this section of if/else statement will only ever return one of these cases, so there is no need to `return`